src/js/actions/pollForProgress.js

```diff
@@ -22,7 +34,7 @@ export default function pollForProgress(polling) {
       .then(json => {
         if(json.status.code < VALIDATED_WITH_ERRORS &&
            json.status.code !== PARSED_WITH_ERRORS){
-             setTimeout(() => poller(dispatch), 1000)
+             setTimeout(() => poller(dispatch), getTimeoutDuration())
```

Comment:
Just a quick question, why use `getTimeoutDuration()` here instead of just `makeDurationGetter()`?

Reply:
it would have to be `makeDurationGetter()()` since it returns a function.

I designed it this way so that the duration getting was testable, on my first pass I just defined `getTimeoutDuration` in the callback, but then it was inaccessible.

Reply:
👍